File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
from collections import Counter

def solution(my_string):
    counts = Counter(my_string)
    answer = []
    
    for i in range(ord('A'), ord('Z') + 1):
        char = chr(i)
        answer.append(counts.get(char, 0))

    
    for i in range(ord('a'), ord('z') + 1):
        char = chr(i)
        answer.append(counts.get(char, 0))
        
    return answer

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    const counts = new Map();
    for (const char of my_string) {
        counts.set(char, (counts.get(char) || 0) + 1);
    }

    const answer = [];

    for (let i = 'A'.charCodeAt(0); i <= 'Z'.charCodeAt(0); i++) {
        const char = String.fromCharCode(i);
        answer.push(counts.get(char) || 0);
    }
    
    for (let i = 'a'.charCodeAt(0); i <= 'z'.charCodeAt(0); i++) {
        const char = String.fromCharCode(i);
        answer.push(counts.get(char) || 0);
    }

    return answer;
}